ext/native/thin3d/GLQueueRunner.cpp

```diff
@@ -142,7 +143,7 @@ void GLQueueRunner::RunInitSteps(const std::vector<GLRInitStep> &steps) {
 			step.create_shader.shader->shader = shader;
 			const char *code = step.create_shader.code;
 			glShaderSource(shader, 1, &code, nullptr);
-			delete[] code;
+			delete[] step.create_shader.code;
```

Comment:
Hmm, should we move this down underneath the error reporting?  Won't this cause a use-after-free currently?

-[Unknown]

Reply:
No need to pretend there's a question, I obviously goofed :) Fixing.